templates/manuscript/show.html.twig

```diff
@@ -18,268 +18,274 @@
+            <table class="table">
+                <thead>
+                    <tr>
+                        <th>Contributor</th>
+                        <th>Role</th>
+                        {% if processNote %}
+                            <th>Note</th>
+                        {% endif %}
+                    </tr>
+                </thead>
+                {% for manuscriptContribution in manuscript.manuscriptContributions|sort((a,b) => a.person.sortableName <=> b.person.sortableName) %}
+                    <tr>
+                        <td>
+                            <a href="{{ path('person_show', {'id': manuscriptContribution.person.id}) }}">{{ manuscriptContribution.person }}</a>
+                        </td>
+                        <td>
+                            <a href="{{ path('manuscript_role_show', {'id': manuscriptContribution.role.id}) }}">{{ manuscriptContribution.role }}</a>
+                        </td>
+                        {% if processNote %}
+                            <td>
+                                {{ manuscriptContribution.note|raw }}
+                            </td>
+                        {% endif %}
 
-{% if isAdmin %}
-{% include 'partial/toolbar.html.twig' with {'edit': 'manuscript_features', 'id': manuscript.id } %}
-{% endif %}
+                    </tr>
+                {% endfor %}
+            </table>
+        {% endif %}
+    {% endset %}
+
+    {% set contentsContent %}
+        {% if isAdmin %}
+            {% include 'partial/toolbar.html.twig' with {'edit': 'manuscript_contents', 'id': manuscript.id } %}
+        {% endif %}
 
-{% if manuscript.getManuscriptFeatures(true)|length > 0 %}
-<table class="table">
-    <thead>
-        <tr>
-            <th>Feature</th>
-            <th>Note</th>
-        </tr>
-    </thead>
-    <tbody>
-        {% for manuscriptFeature in manuscript.getManuscriptFeatures(true) %}
-        <tr>
-            <td>
-                <a href="{{ path('feature_show', {'id': manuscriptFeature.feature.id}) }}">
-                    {{ manuscriptFeature.feature.label }}
-                </a>
-            </td>
-            <td>{{ manuscriptFeature.note|raw }}</td>
-        </tr>
-        {% endfor %}
-    </tbody>
-</table>
-{% endif %}
-{% endset %}
+        {% if manuscript.manuscriptContents|length > 0 %}
+
+            {% set showPrintSource = manuscript.manuscriptContents|filter(m => m.printSource is not null)|length > 0 %}
+            <table class="table">
+                <thead>
+                    <tr>
+                        <th>First Line</th>
+                        <th>Context</th>
+                        {% if showPrintSource %}
+                            <th>Print Source</th>
+                        {% endif %}
+                    </tr>
+                </thead>
+                <tbody>
+                    {% for manuscriptContent in manuscript.manuscriptContents|sort((a,b) => a.content.firstLine <=> b.content.firstLine) %}
```

Comment:
Not requesting a change here, but we should probably check with MVM to see if they'd prefer sorted by firstLine or by date (though I suppose firstLine is more reliable)